src/runtime/template_set.ts

```diff
@@ -22,6 +23,7 @@ export class TemplateSet {
   dev: boolean;
   rawTemplates: typeof globalTemplates = Object.create(globalTemplates);
   templates: { [name: string]: Template } = {};
+  getRawTemplate?: (s: string) => Element | Function | string | null;
```

Comment:
I think returning undefined instead of null would make more sense, it allows avoiding an explicit null return.